kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/source/reader/ReaderManager.scala

```diff
@@ -76,12 +76,12 @@ class ReaderManager(
             case Some(value) =>
               // The index of -1 means no record. It's an unfortunate state introduced by the readers keeping track of the current records
```

Comment:
thy not try to put this comment into `transposeRecordIdxForLog` function? (or maybe even Scaladoc just for us?)

Reply:
Good point, I have tidied up the comments so the `transposeRecordIdxForLog` explains all.